### PR TITLE
virt-install, virt-sparsify: add page

### PR DESCRIPTION
**Pull request by bergentroll**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/virt-install.md
Comment (sbrl) [inline pages/common/virt-install.md:4]: I think the linting error is here. I can certainly see why a lowercase letter would be preferred here though. What about this instead?
```suggestion
> Create new virtual machines with libvirt, installing the operating system automatically.
> More information: <https://virt-manager.org/>.
```

File: pages/common/virt-install.md
Comment (sbrl) [inline pages/common/virt-install.md:6]: Perhaps this will read better?
```suggestion
- Create a virtual machine with 1 GiB RAM and 12 GiB storage and install Debian:
```

File: pages/common/virt-install.md
Comment (sbrl) [inline pages/common/virt-install.md:8]: Very minor thing really - just for consistency with other pages :-)
```suggestion
`virt-install --memory {{1024}} --disk path={{path/to/image.qcow2}},size={{12}} --cdrom {{path/to/file/debian.iso}}`
```

File: pages/common/virt-sparsify.md
Comment (sbrl) [inline pages/common/virt-sparsify.md:5]: Same deal here as with the previous page. What about this?
```suggestion
> Make virtual machine drive images thin-provisioned.
> Use only for offline machines to avoid data corruption.
> More information: <https://libguestfs.org/>.
```

File: pages/common/virt-sparsify.md
Comment (sbrl) [inline pages/common/virt-sparsify.md:7]: Typo, and maybe this will read better :-)
```suggestion
- Create a sparsified compressed image without snapshots from an unsparsified image:
```

File: pages/common/virt-sparsify.md
Comment (sbrl) [inline pages/common/virt-sparsify.md:9]: More consistency
```suggestion
`virt-sparsify --compress {{path/to/image.qcow2}} {{path/to/image_new.qcow2}}`
```

File: pages/common/virt-sparsify.md
Comment (sbrl) [inline pages/common/virt-sparsify.md:11]: Perhaps this will read better?
```suggestion
- Sparsify an image in-place:
```

File: pages/common/virt-sparsify.md
Comment (sbrl) [inline pages/common/virt-sparsify.md:13]: Consistency again
```suggestion
`virt-sparsify --in-place {{path/to/image.img}}`
```
....unless images _have_ to be located in the directory you've specified there in order for it to work properly - in which case ignore this comment.

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page, @bergentroll! I've left some comments below for you to review.

Great additions, by the way! We don't have nearly enough pages about virtual machine management tools on the CLI.

File: pages/common/virt-install.md
Comment (bergentroll) [reply pages/common/virt-install.md:4]: Well, that is a good fix for linter. But I have a couple of comments:
- Utility does not actually _intall_ OS, it only boots an image, that is the point.
- Home page actually does not provide a lot of information, while [redhat.com page](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/virtualization_host_configuration_and_guest_installation_guide/sect-virtualization_host_configuration_and_guest_installation_guide-guest_installation-creating_guests_with_virt_install) does, but I am not sure it is an appropriate link.

File: pages/common/virt-install.md
Comment (bergentroll) [reply pages/common/virt-install.md:6]: And about installer starting again.

File: pages/common/virt-install.md
Comment (bergentroll) [reply pages/common/virt-install.md:8]: I was afraid of long-long line, but I am fully agreed if you find it OK. And an absolute path came from [qemu-img](../blob/master/pages/common/qemu-img.md) page.

File: pages/common/virt-sparsify.md
Comment (bergentroll) [reply pages/common/virt-sparsify.md:5]: I'd prefer to keep a libguestfs mention, cause it could help to match the utility with a distro package. But it is up to you.

File: pages/common/virt-sparsify.md
Comment (bergentroll) [reply pages/common/virt-sparsify.md:7]: Oh, indeed, thank you!

**Comment by bergentroll** [comment]
Well, I make few changes due to @sbrl comments (thank you, **sbrl**!). I made few commits with web and a couple on my host, and CLA sees now two committers. I have now idea why, because I see identical author line in `git log`. I believe, it will be eliminated with squashing.

**Review by sbrl — APPROVED** [review]
Thanks, @bergentroll!

Ah, it's the email address that's the issue here. Let me see if I can correct the issue.

**Comment by sbrl** [comment]
Ah, I've fixed the issue @bergentroll, but it looks like you're not allowing edits from maintainers on your branch there - so I can't push the fix!

To do it yourself, you'll need to open a terminal / command prompt and execute the following:

```bash
cd "path/to/tldr_repository_here";
git rebase -i HEAD~2 -x "git commit --amend --author 'Anton Karmanov <bergentroll@insiberia.net>' --no-edit";
# A text editor will open - save and close it
git push --force
```

Once we've got a 2nd approval, we can merge this :-)

**Comment by bergentroll** [comment]
@sbrl, yeah, that was ungood decision. Thank you, fixed!

**Comment by bergentroll** [comment]
@sbrl, also could you please advise how to see the issue in such cases? In git log I saw only "Author: Anton Karmanov <bergentroll@insiberia.net>".

**Review by navarroaxel — APPROVED** [review]

**Comment by sbrl** [comment]
Hey there, @bergentroll! Personally I used [tig](https://github.com/jonas/tig), which is a text-mode user interface for git. Apparently there are multiple authorship fields for commits (haven't read up on it, so  I'm unsure what they all are/for), and it wasn't the one shown by  `git log` that caused the issue. Very odd indeed!

**Pull request merged**
